pkg/azuredisk/nodeserver_test.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
2019
BTW, could you work out a tool to add this statement to all source code under this project? Thanks.

Reply:
Thanks, I will fix it.